mapping/jsonld-contexts/mapping.jsonld

```diff
@@ -1,23 +1,23 @@
 {
 	"@context": {
-		"Mapping": "https://ontology.eglobalmark.com/Mapping",
```

Comment:
guess it will probably break the MQTT bridge. do we really want to add this burden?

Reply:
No we don't :)
If I leave Mapping and Bridge to the old expanded will it be fine ? (they don't have any impact on the flow mapping)

Reply:
I really only need to change "attributesToMap", "attribute", "context" and "hasTargetEntity"

Reply:
good news (for you), it should not have any impact on MQTT Bridge

Reply:
No impact if I revert Mapping and Bridge or can I go like that ?

Reply:
the full modification